### ISSUE TITLE
This repository is not referenced from anywhere and hard to find

### ISSUE THREAD

**Issue by sergeykad** [eclipse-ee4j/eclipselink#268]
1. Searching in Google for Eclipselink repository returns https://github.com/eclipse/eclipselink.runtime as a first result. The repository seems legit on the first glance, until you check date of the last commit. There is no any other indication that is is not uses anymore or link to this repository.
2. I was unable to find any references to this repository on eclipse.org
3. There is a bug tracker on eclipse.org that allows opening new bugs for Eclipselink, but it seems that nobody check them anymore. There is no clear indication for it, but I did not notice much activity there.

The bottom line is that although this repository is public, many potential contributors may miss it and create pull requests on https://github.com/eclipse/eclipselink.runtime or open bugs on https://bugs.eclipse.org/bugs/describecomponents.cgi?product=EclipseLink
😕

**Comment by dazey3** [comment]
1. We can't do much about google query results, but maybe we should shutdown https://github.com/eclipse/eclipselink.runtime. It's just a mirror, right? Is there a reason we want to keep it open?

2. This looks like something @lukasj should look at. Maybe https://www.eclipse.org/eclipselink/ should have a section like:
```
    Looking to contribute? EclipseLink is Open Source!
    EclipseLink on GitHub: https://github.com/eclipse-ee4j/eclipselink

```

3. Bugs can still be opened through https://bugs.eclipse.org/bugs/describecomponents.cgi?product=EclipseLink, which is perfectly fine. I don't think that has been abandoned. Noticing a lack of activity is just reality :/ we only have 4/5 active contributors (more or less).

**Comment by lukasj** [comment]
Wrt 2, there is https://projects.eclipse.org/projects/ee4j.eclipselink/developer

**Comment by sergeykad** [comment]
Thanks for clarifying. 
Sorry to hear that there is only 4/5 active contributors for such a big projects. I was under impression that with backing of IBM and Oracle it has much more active developers.

**Comment by dazey3** [comment]
@lukasj I'm not sure I understand your comment. I think the point is that https://projects.eclipse.org/projects/ee4j.eclipselink, and similar sites for EclipseLink, don't have links anywhere to this GitHub. Maybe I am missing the link on https://projects.eclipse.org/projects/ee4j.eclipselink/dazey3

**Comment by lukasj** [comment]
if you look at https://projects.eclipse.org/projects/ee4j.eclipselink/developer then the very first repo listed on that page is "https://github.com/eclipse-ee4j/eclipselink" and the Browse repository link next to this takes you to github

**Comment by dazey3** [comment]
@lukasj Oh, I see. I thought your meant something else... but I see it now. Thanks! Also, what are your thoughts wrt 1?

**Comment by sergeykad** [comment]
Maybe I am missing something, but is not this page is the main EclipseLink page http://www.eclipse.org/eclipselink/ ?
I don't see reference to GitHub there, although it allows downloading sources as a ZIP file.

**Comment by dazey3** [comment]
@sergeykad There is a link at the top of that page, in the `More` drop down, called `About this project`. This should redirect you to the https://projects.eclipse.org page. This page then has a tab called `Developer Resources` which lists the GitHub URL... Before you say it, I think that is kind of buried too...

My concern is that also under the `More` drop down is the option `Learn how to contribute`, which doesn't seem to reference the GitHub repository. There is a link on that page to https://wiki.eclipse.org/EclipseLink/Development/Process/Git, but it's full of 404 links and needs to be updated. Is it possible we can start porting this stuff to https://github.com/eclipse-ee4j/eclipselink/wiki and deprecate the https://wiki.eclipse.org/EclipseLink/Development stuff?

To be fair, when we change repos and update processes, its easy to forget to update documentation and wikis.

**Comment by dazey3** [comment]
Interesting, so do we have two landing pages that list all the repositories?:
    1. http://git.eclipse.org/c/eclipselink/
    2. https://projects.eclipse.org/projects/ee4j.eclipselink/developer

I feel like the `Check-out the source code` link should direct to https://projects.eclipse.org/projects/ee4j.eclipselink/developer instead. can we not setup a GitHub repo for `www.eclipse.org/eclipselink.git`? That repo would be a good location for discussion like what we are having.

**Comment by dazey3** [comment]
I see that for 531036, the original plan was to remove the mirror repo eventually, but we never got around to it. I made a comment to see if we can do that now.

**Comment by Flugtiger** [comment]
Just my two cents: I did not find the github repository on the first try neither. There currently seems to be no link from the main eclipselink website (https://www.eclipse.org/eclipselink/) to this github repo. I was also not aware of the newer projects.eclipse.org website.

**Comment by lukasj** [comment]
note that AFAICT projects.eclipse.org entry for eclipselink is there since 2007. The only change in 2018 here was the renaming  from rt.eclipselink to ee4j.eclipselink. That is not to justify why this is not fixed yet...

**Comment by sergeykad** [comment]
There is a link as @dazey3 mentioned above, but it is not easy to find: https://projects.eclipse.org/projects/ee4j.eclipselink/developer

**Comment by laeubi** [comment]
I also can confirm that it was hard to notice that for example bugzilla is not used any more (at leas no one even seems to care to left a note on questions/issues for over a year).

**Comment by lukasj** [comment]
> 1. Searching in Google for Eclipselink repository returns https://github.com/eclipse/eclipselink.runtime as a first result

not true, at least not in my case

> 2. I was unable to find any references to this repository on eclipse.org

https://projects.eclipse.org/projects/ee4j.eclipselink/developer lists all repositories related to the eclipselink project, it is hosted on eclipse.org and should be found by using search for the project name

> 3. There is a bug tracker on eclipse.org that allows opening new bugs for Eclipselink, but it seems that nobody check them anymore. There is no clear indication for it, but I did not notice much activity there.

BOTH trackers are checked for incoming issues regularly. Since there are >1000 open issues, those having votes do have higher priority

**Comment by laeubi** [comment]
> and should be found by using search for the project name
https://www.eclipse.org/eclipselink > More > Check out the source code > https://git.eclipse.org/c/eclipselink/

> BOTH trackers are checked [...] regularly [.. ] those having votes do have higher priority
Highest voted Bug has last activity 2019-03-20 (over one year), next most voted one even more then two year (2018-03-27)

**Comment by lukasj** [comment]
@laeubi so you prefer bug fixes over required Jakarta EE related work. Ok, noted. Wrt those links - is it that difficult to file a bug just for that or define _exactly_ what _anywhere_ means?

**Comment by laeubi** [comment]
In general bugfixes should be preferred over new functions, right nerveless I never wrote "anywhere" (that was the original author @sergeykad around 1,5years ago and probably things has changed since there.

So I'm just a bit confused about aggressive  response (maybe just my interpretation) of your response, I just wanted to point out that even for me (who has contributed already to different Eclipse and other OSS projects) to find out that repository was moved. Then you closed the ticket with just a reference to the initial text and I felt to clarify why I still find it was an issue.

**Comment by lukasj** [comment]
@laeubi Sorry if it feels aggressive, it is not supposed to be. The fact is, that there are higher priority tasks which have to be done first - and this is really not about new functions/features, it is just about the move to `jakarta` namespace and it is not something project can influence. But contributions are welcomed.

The way the project deals with two bug trackers can be seen ie in #59. There also exists mailing list, where questions can be asked - it is linked from the repo's readme; I for one do not think that issue tracker is the right place for questions.

> nerveless I never wrote "anywhere" (that was the original author @sergeykad around 1,5years ago and probably things has changed since there.

correct, you did not and this should have been closed already long time ago, sorry. Why did you hijack the issue for different problem then?

**Comment by laeubi** [comment]
I will open an new ticket then and just tried to check if there is an existing one.
